Refresh

talimarcus opened the pull request:
Design refresh (spec here: https://ge.ent.box.com/notes/185167242781?s=sk8uem5vb8x4sd2c3sywxldbfx3qfktl) 

Note: Wade confirmed that it's okay to use to use a popover instead of a dropdown (as illustrated in the spec).

katemenkhaus commented on sass/px-login.scss:
```diff
@@ -39,33 +39,37 @@ $inuit-enable-btn--bare : true;
+.login-button--chevron-icon {
+  width: 12px;
+  min-width: 12px;
+  margin-left: 5px;
```
with margin's we generally use $inuit-base-spacing-unit.

katemenkhaus commented on sass/px-login.scss:
```diff
@@ -39,33 +39,37 @@ $inuit-enable-btn--bare : true;
-    color: var(--px-login-color, $gray13);
+.login-button--user-icon {
+  min-width: 30px;
+  height: 30px;
```
I would also use $inuit-base-spacing-unit here because you want it to be dependent on the font size

Menkhaus-ge commented:
Did you double check that you needed all the imports in the sass?

Menkhaus-ge commented:
also Martin mentioned that sense our components aren't getting qa we need to make sure we have enough unit tests.

talimarcus commented:
@Menkhaus-ge:

- updated to use $inuit-base-spacing-unit and calculateRem()
- removed unnecessary sass imports
- checked the test coverage, looks good to me

talimarcus commented:
@Menkhaus-ge one last review please? :)

Menkhaus-ge commented:
Looks good
